app/Http/Controllers/SessionController.php

```diff
@@ -59,28 +59,25 @@ public function __construct(ConfigFunctions $configFunctions, GitHubFunctions $g
 	 */
 	public function init(): array
 	{
-		$logged_in = AccessControl::is_logged_in();
+		$logged_in = Authorization::check();
 
 		// Return settings
 		$return = [];
 
 		// Check if login credentials exist and login if they don't
-		if (AccessControl::noLogin() === true || $logged_in === true) {
+		if (Authorization::isAdminNotConfigured() || $logged_in === true) {
 			// we set the user ID (it is set to 0 if there is no login/password = admin)
-			$user_id = AccessControl::id();
+			$user_id = Authorization::idOrFail();
```

Comment:
I am not sure, if this one is correct. What happens if no admin is configured? Does this method return `0` as the comment says or does it fail?